src/applications/vaos/components/calendar/CalendarWidget.jsx

```diff
@@ -144,8 +237,8 @@ export default function CalendarWidget({
   maxSelectionsError = "You've exceeded the maximum number of selections",
   minDate,
   onChange,
-  onClickNext,
-  onClickPrev,
+  onClickNext: onNextMonth,
```

Comment:
It looks like this just renames the prop when using inside the component. Ideally, we would change these prop names for consumers of this component, too, and update the three pages that use the CalendarWidget.

Reply:
Makes sense, fixed.